src/components/nav-bar/nav-bar.tsx

```diff
@@ -35,10 +35,10 @@ export const NavBar: React.FunctionComponent<NavBarProps> = (props: NavBarProps,
                 <div className='nav-bar__version'>{props.version && props.version()}</div>
                 {(props.items || []).map((item) => (
                     <Tooltip content={item.title} placement='right' arrow={true} key={item.path + item.title}>
-                        <div className={classNames('nav-bar__item', { active: isActiveRoute(locationPath, item.path) })}
+                        <button type="button" className={classNames('nav-bar__item', { active: isActiveRoute(locationPath, item.path) })}
```

Comment:
I think this can actually just use a `<Link>` element from `react-router`, which would also solve #166